src/status_im/contexts/wallet/bridge/bridge_to/view.cljs

```diff
@@ -8,27 +8,38 @@
     [status-im.contexts.wallet.bridge.bridge-to.style :as style]
     [status-im.contexts.wallet.common.account-switcher.view :as account-switcher]
     [status-im.contexts.wallet.common.utils :as utils]
+    [status-im.contexts.wallet.common.utils.networks :as network-utils]
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
 (defn- bridge-token-component
   []
   (fn [{:keys [chain-id network-name]} token]
-    (let [network         (rf/sub [:wallet/network-details-by-chain-id chain-id])
-          currency        (rf/sub [:profile/currency])
-          currency-symbol (rf/sub [:profile/currency-symbol])
-          balance         (utils/calculate-total-token-balance token [chain-id])
-          crypto-value    (utils/get-standard-crypto-format token balance)
-          fiat-value      (utils/calculate-token-fiat-value
-                           {:currency currency
-                            :balance  balance
-                            :token    token})
-          fiat-formatted  (utils/get-standard-fiat-format crypto-value currency-symbol fiat-value)]
+    (let [theme                       (quo.theme/use-theme)
+          network                     (rf/sub [:wallet/network-details-by-chain-id chain-id])
+          currency                    (rf/sub [:profile/currency])
+          currency-symbol             (rf/sub [:profile/currency-symbol])
+          networks                    (rf/sub [:wallet/network-details])
+          balance                     (utils/calculate-total-token-balance token [chain-id])
+          crypto-value                (utils/get-standard-crypto-format token balance)
+          fiat-value                  (utils/calculate-token-fiat-value
+                                       {:currency currency
+                                        :balance  balance
+                                        :token    token})
+          fiat-formatted              (utils/get-standard-fiat-format crypto-value
+                                                                      currency-symbol
+                                                                      fiat-value)
+          token-networks              (network-utils/network-list token networks)
+          token-networks-ids          (mapv #(:chain-id %) token-networks)
+          token-networks-ids-set      (set token-networks-ids)
+          token-available-on-network? (contains? token-networks-ids-set chain-id)]
```

Comment:
do you really need `token-networks`, `token-networks-ids`, `token-networks-ids-set`?

seems like this should all be done in the sub for `token-available-on-network?` ??

Reply:
or at least a util for that 🤔

Reply:
imo the let binding should just include the minimal data needed by the view's props. Other data can be taken out to utils or subs

Reply:
Yep, simplified the approach